canonical-web-and-design/jaas.ai
# Search icon cleanup

## Done

- Removed unused default icon files.
- Remove icon checking/default icon code (default icons are returned by the charmstore).
- Dedupe icon html in the template.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Do some searches. There should be icons for charms/bundles.
